### PR TITLE
Ignite links

### PR DESCRIPTION
**Pull request by nirmitparikh8**
I made 4 main ignite event pages. Added Buttons on the main pages going to the different pages of the event. Updated Latest Works. Removed YC finder.

### CODE REVIEW

File: functions/index.js [hunk 1]
```diff
@@ -32,31 +32,58 @@ app.get("/workshops", function(request, response) {
...
   response.render("pages/fuse17", {
-    name: "fuse"
+    name: "fuse17"
   });
 });
+
 app.get("/fuse19", function(request, response) {
   response.render("pages/fuse19", {
-    name: "fuse"
+    name: "fuse 2019"
```
Comment (Dhvani35729) [inline functions/index.js:74]: Why is there a space here? I think the convention has been no spaces.

Comment (nirmitparikh8) [reply functions/index.js:74]: wdym by convention?

Comment (Dhvani35729) [reply functions/index.js:74]: Convention meaning what does the existing code do

**Comment by Dhvani35729** [comment]
> _No description provided._

Please provide a description of changes made

File: functions/index.js (hunk 1 above)
Comment (nirmitparikh8) [reply functions/index.js:74]: IDK why there is a space. How does that impact the code though?

Comment (KrishKrosh) [reply functions/index.js:74]: It is better to use the same conventions throughout the code to reduce confusion

```suggestion
    name: "fuse19"
```

Comment (nirmitparikh8) [reply functions/index.js:74]: Okie, I will change that

**Comment by nirmitparikh8** [comment]
>I fixed the pictures, latest work blocks, removed fuse 2019 from media, removed stempower 2019 event, added fuse 2019 to latest works, and fixed the margins on STEMpower 2019. Couldn't fix JUnto Support and Ignite on the home page. I don't know how to center it. Can someone help? I committed a couple of changes recently, tell me if you got them.

File: functions/views/pages/stempower.ejs [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+			}
+			</style>
+		<div class="overlay"></div>
+		<div class="gtco-container">
+			<div class="row">
+				<div class="col-md-8 col-md-offset-2 text-center">
+					<div class="display-t">
+						<div class="display-tc">
+							<h1 class="animate-box" data-animate-effect="fadeInUp">STEMpower</h1>
+							<h2 class="animate-box" data-animate-effect="fadeInUp">STEMpower is a branch of YouthComputing that holds mini presentations at elemntary schools. This branch showcases and exposes cool technologies, exposing the tech world to the younger students.</h2>
```
Comment (Dhvani35729) [inline functions/views/pages/stempower.ejs:45]: Spelling error. Says: "at elemntary schools"

File: functions/views/pages/fuse.ejs [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+							<div class="item">
+								<img src="images/fuse2.jpg" width="100%">
+							</div>
+								<p></p>
+								<p></p>
+								<p><a href="fuse19" class="round">Fuse 2019</a></p>
+								<h1></h1>
+
+						<div class="col-md-12">
+							<h3>Drones at Fuse 2017</h3>
```
Comment (Dhvani35729) [inline functions/views/pages/fuse.ejs:70]: This div is too close to the top one. Add a margin top

File: functions/views/pages/fuse19.ejs [hunk 4]
```diff
@@ -15,50 +15,66 @@
...
 		<div class="gtco-container">
 				
 			<div class="row">
 				<div class="col-md-8 col-md-offset-2 text-center gtco-heading">
 					<h2>Fuse 2019</h2>
-					<p>YouthComputing is hosting Fuse 2019 at Westwood Community High School on November 25. Fuse is an open house showcasing various technologies and promotes science, technology, engineering, and mathematics.</p>
-				<h3>Pictures from the event: </h3>
-				<a href="https://woodbuffalo.snapd.com/events/view/1104996">https://woodbuffalo.snapd.com/events/view/1104996</a>
-				<a href="https://www.dropbox.com/sh/94n5qcy9r6pcxum/AAAH7fc36wikvLBEzGBa4FdAa?dl=0">https://www.dropbox.com/sh/94n5qcy9r6pcxum/AAAH7fc36wikvLBEzGBa4FdAa?dl=0</a>
+					<p>YouthComputing is hosting Fuse 2019 at Westwood Community High School on December 21. Fuse is an oppurtunity for youth to showcase various technological inventions and it also promotes science, technology, engineering, art and mathematics.</p>
```
Comment (Dhvani35729) [inline functions/views/pages/fuse19.ejs:58]: change is hosting to hosted

File: functions/views/pages/fuse19.ejs [hunk 5]
```diff
@@ -15,50 +15,66 @@
...
 				<div class="col-md-12">
-					<h3>Information about the Different Booths</h3>
+				<div class="col-md-8 col-md-offset-2 text-center gtco-heading">
+					<h3>Below are some of the pictures from the event: </h3>
+					<div class="item">
+						<img src="images/fuse4.jpg" width="100%" alt="Booths">
+					</div>
+					<p></p>
+					<p></p>
+					<p> <a href="https://drive.google.com/drive/folders/13lqEoBKIoQVqhJMRkmJdQJf0vyRAT4l3?usp=sharing" class="round">More Pictures From Fuse 2019</a></p>
```
Comment (Dhvani35729) [inline functions/views/pages/fuse19.ejs:70]: this link is to old drive, update the link

File: functions/views/pages/fuse17.ejs [hunk 6]
```diff
@@ -16,52 +16,67 @@
...
-				<h3>Pictures from the event: </h3>
-				<a href="https://woodbuffalo.snapd.com/events/view/1104996">https://woodbuffalo.snapd.com/events/view/1104996</a>
-				<a href="https://www.dropbox.com/sh/94n5qcy9r6pcxum/AAAH7fc36wikvLBEzGBa4FdAa?dl=0">https://www.dropbox.com/sh/94n5qcy9r6pcxum/AAAH7fc36wikvLBEzGBa4FdAa?dl=0</a>
-				</div>
+	<div class="row">
+		<div class="col-md-8 col-md-offset-2 text-center gtco-heading">
+			<h1></h1>
+			<div class="col-md-12">
+				<h2>Fuse 2017</h2>
+				<p>YouthComputing is hosting Fuse 2017 at</p>
```
Comment (Dhvani35729) [inline functions/views/pages/fuse17.ejs:57]: change is hosted to hosted

File: functions/views/pages/ccd.ejs [hunk 7]
```diff
@@ -16,788 +16,71 @@
...
+        <div class="overlay"></div>
 		<div class="gtco-container">
 			<div class="row">
 				<div class="col-md-8 col-md-offset-2 text-center">
 					<div class="display-t">
 						<div class="display-tc">
 							<h1 class="animate-box" data-animate-effect="fadeInUp">Community Code Day</h1>
-							<h2 class="animate-box" data-animate-effect="fadeInUp">2018 | Arts & Culture</h2>
-							<a href="https://communitycodeday.com/"><h2>https://communitycodeday.com/</h2></a>
+							<h2 class="animate-box" data-animate-effect="fadeInUp">Community Code Day is Fort McMurray's first community technology conference. People can participate in a hackathon, and / or they can choose from a selection of speakers and workshops to learn how to use technology as a beneficial factor in their endeavours.</h2>
```
Comment (Dhvani35729) [inline functions/views/pages/ccd.ejs:45]: change first to annual

**Comment by Dhvani35729** [comment]
@nirmitparikh8 , you should add the pictures from CryptoChicks workshop to workshops page

**Comment by nirmitparikh8** [comment]
> @nirmitparikh8 , you should add the pictures from CryptoChicks workshop to workshops page

I can't find the pictures for the cryptochicks workshop

File: functions/views/pages/fuse19.ejs (hunk 5 above)
Comment (nirmitparikh8) [reply functions/views/pages/fuse19.ejs:70]: Admin needs to enable link sharing out of the YC organization

**Comment by nirmitparikh8** [comment]
> @nirmitparikh8 , you should add the pictures from CryptoChicks workshop to workshops page

Done

File: functions/views/pages/fuse19.ejs (hunk 5 above)
Comment (nirmitparikh8) [reply functions/views/pages/fuse19.ejs:70]: Talked with Krish and we cannot share outside of our organization, so I just kept the old link

**Comment by Dhvani35729** [comment]
+1

**Pull request merged**